src/java.base/share/classes/java/lang/module/ModuleDescriptor.java

```diff
@@ -1312,7 +1312,7 @@ private ModuleDescriptor(String name,
      * Creates a module descriptor from its components.
      * The arguments are pre-validated and sets are unmodifiable sets.
      */
-    ModuleDescriptor(String name,
+    private ModuleDescriptor(String name,
                      Version version,
                      Set<Modifier> modifiers,
                      Set<Requires> requires,
```

Comment:
I assume you'll fix the alignment if this goes ahead.

Reply:
Thanks! Yes, that's good feedback. There are some non-API changes that I'd like to make, such as this and updating the comment for the `InterfaceAddress` constructor.

I prefered to make the initial PR draft contain just the CSR changes, such that PR would be useful and clean as a Specification change, since the diff is a bit noisy.

I'll keep your review comment here unresolved for now and get back to it pending CSR approval.

Reply:
For the record:

It was not completely clear to me whether restricting these two package-private constructors to private is considered a specification change in the CSR sense. I opted to point them out in the CSR, just to make the change abundantly clear.

Reply:
This constructor is part of the public API: package-private is not accessible, and it does not appear in the API docs: https://docs.oracle.com/en/java/javase/23/docs/api/java.base/java/lang/module/ModuleDescriptor.html